Changelog.md

```diff
@@ -1,26 +1,12 @@
 # Changelog
 
-## Unreleased new version
```

Comment:
Do we want to keep that at the top? of the file? I don't really care either way, but that how I've done it in Roto.

Reply:
I always strip it for the release, and add it back in the post-release commit on main.